<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WITFU</class>
 <widget class="QMainWindow" name="WITFU">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WITFU</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>831</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Générer code barre</string>
     </attribute>
     <widget class="QPushButton" name="pB_generate_EAN13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>241</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>générer le code barre</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lE_URLserial">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>40</y>
        <width>261</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cB_URLtype">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="EAN13" name="ean13_view" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pB_saveImage">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>sauvegarder le code barre</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>251</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Numéro de série</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_code">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lE_URLref">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>271</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>251</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Ref</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="debugGen">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>510</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Debug</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_etiquette">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>671</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Scanner les URLs</string>
     </attribute>
     <widget class="QPushButton" name="pB_scan_device">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>scan device</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="conf">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>601</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="debugScan">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>520</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Debug</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pB_sauvConf">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>311</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Sauvegarder la conf</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lE_conf">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>60</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>859</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>EAN13</class>
   <extends>QWidget</extends>
   <header>ean13.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
